iclr2016(old scripts)/20160114-demo/web-field-workshop.html

```diff
@@ -1,8 +1,4 @@
 <html>
-  <head>
-    <!--<script src="https://code.jquery.com/jquery-1.10.1.min.js"></script>-->
-    <script src="https://cdn.jsdelivr.net/lodash/3.5.0/lodash.compat.min.js"></script>
```

Comment:
This is part of the reason I'd like to move away from the webfields being full HTML files. Since they aren't being loaded in an iframe (that would be a whole other set of headaches), we really only need them to be .js files, which are easier to sanitize, validate, and execute in a sandbox environment where we control what global variables are accessible.